src/edu/wright/cs/raiderplanner/controller/MenuController.java

```diff
@@ -190,22 +188,17 @@
 	@FXML
 	private HBox exportCalBox;
 
-	//chat variables
-	private final BorderPane mainPane = new BorderPane();
+	// Chat Variables
+	private static final BorderPane mainPane = new BorderPane();
 	private final GridPane firstPane = new GridPane();
-	private final GridPane userMessagePane = new GridPane();
-	private final HBox spacingBox = new HBox();
 	private TextField tfName = new TextField("");
 	private TextField tfHost = new TextField("");
-	private TextField tfMessageToSend = new TextField();
-	private TextArea msgArea = new TextArea();
 	private final Label name = new Label("Name:");
 	private final Label host = new Label("Host:");
 	private final Button submitButton = new Button("Submit");
-	private final Button sendButton = new Button("Send");
-	private boolean calendarOpen = false; //Used to monitor status of calendar (open or closed)
 
-	private String userName;
+	private boolean calendarOpen = false; //Used to monitor status of calendar (open or closed)
+	private static String userName;
```

Comment:
Why was this variable made static? It does not make sense that all instances of the class and even no instance of the class would have the same username associated.